web/Weblate-server.qmd

```diff
@@ -205,26 +206,68 @@ rebase refs/remotes/origin/trunk: command returned error: 1
 Weblate will provide a suggestion on how to fix, but in short:
 
 1. SSH to the Weblate server
-2. Attach the Docker container:
+2. Make a backup on the Git repo before messing with the rebase process:
+
+    ```sh
+    tar -zcvf /root/base-r-gui-repo-$(date '+%Y-%m-%d').tar.gz \
+      /var/lib/docker/volumes/weblate-docker_weblate-data/_data/vcs/r-project/base-r-gui
+    ```
+
+3. Attach the Docker container (this image is offered by `weblate`):
 
     ```sh
     sudo docker exec -ti weblate-docker_weblate_1 bash
     ```
 
-3. Enter the git folder:
+4. Enter the git folder:
 
     ```sh
     cd app/data/vcs/r-project/base-r-gui
     ```
 
-4. Pull from SVN and start the rebase process to see the actual errors:
+    When looking for this folder outside of Docker, you can find it at
```

Comment:
this is the same folder referenced in item 2 above right?

Reply:
yes

Reply:
OK, reiterated that for clarity